locales/de/common.json

```diff
@@ -43,7 +43,7 @@
   "Beta": "Beta",
   "Add": "Hinzufügen",
   "Number of Mods enabled": "Anzahl aktiver mods",
-  "Search for games": "Nach Spielen suchen",
+  "Search for Games": "Nach Spielen suchen",
```

Comment:
Sorry, but that's not correct. Games would only be capitalised at this point if it was a title, but it's not, it's a tooltip/button label.